selfdrive/modeld/models/driving.cc

```diff
@@ -133,13 +133,19 @@ void fill_path(cereal::ModelData::PathData::Builder path, const float * data, bo
   float poly_arr[POLYFIT_DEGREE];
   float std;
   float prob;
+  float valid_len;
 
+  valid_len =  data[MODEL_PATH_DISTANCE*2];
   for (int i=0; i<MODEL_PATH_DISTANCE; i++) {
     points_arr[i] = data[i] + offset;
-    stds_arr[i] = softplus(data[MODEL_PATH_DISTANCE + i]);
+    if (i < valid_len) {
```

Comment:
Is there a minimum lower sanity bound on this? Concerned it'll be <= 4, and fit a crazy third order polynomial.

Reply:
Hmmmm, during training the minimum for path is 5. However there may not be a minimum for lanelines. I'll add a minimum.

Reply:
Always good to enforce the minimum on the model outputs here as well, it's always possible to get weird outputs from metaparams like this.

Reply:
yeah I'll add that